### ISSUE TITLE
[Feature Request] Transform the questions in optional

### ISSUE THREAD

**Issue by devajmeireles** [pinkary-project/pinkary.com#630]
Since if I did the PR, it would probably be rejected, I'm sending it as a feature suggestion for someone with more credit than me to do it 😄 Maybe, @MrPunyapal .

---

Currently, the questions feature is not optional, but some people, like me, may not want to use it, since Pinkary's initial proposal - I think, was for bio links. 

My suggestion would be to have the possibility of disabling the questions, and only allowing the idea of sharing updates. So, those who are visitors to our profile will not see this text 👇🏻

Without making this optional, the Pinkary will not be interesting for some people who just want to use the platform for bio links.

![CleanShot 2024-09-19 at 10 30 59](https://github.com/user-attachments/assets/d607e72b-0053-4d4e-a8d8-11cd6f8055e9)

**Comment by MrPunyapal** [comment]
It's a good Idea 💡 

You may proceed 🫡

cc: @nunomaduro

**Comment by rahat1994** [comment]
Suppose there is a toggle button on the settings page to enable/disable that feature for individual profiles that would do it. right?

**Comment by MrPunyapal** [comment]
> Suppose there is a toggle button on the settings page to enable/disable that feature for individual profiles that would do it. right?

Yes, maybe a checkbox. It should be enabled by default. And good tests should make sure they can share updates but others cannot ask them questions.

**Comment by rahat1994** [comment]
My Plan:

add a new column to the `users` table

`$table->boolean('questions_enabled')->default(true);`

and then reflect this change on the profile page. If you want to name the column anything else just let me know and can you please assign it to me.

**Comment by rahat1994** [comment]
Hey @MrPunyapal  do I need to do 100% test coverage because Its 99.5%? Files that fail test coverage.

1. Tests\Unit\Jobs\UpdateUserAvatarTest -> skipped due to windows machine.
2. app\Livewire\Questions\CreateTest.php

**Comment by MrPunyapal** [comment]
You ignore that stuff 👌

**Comment by rahat1994** [comment]
#687
